stingray/gti.py

```diff
@@ -393,3 +395,47 @@ def append_gtis(gti0, gti1):
             'exclusive.')
 
     return np.concatenate([gti0, gti1])
+
+
+def join_gtis(gti0, gti1):
+    """Merge two overlapping GTIs
+
```

Comment:
I also corrected the function. It's a union that I meant, but for some reason I implemented a sort of mixed operation, where I appended the intervals not in common and intersected the ones in common. The new version is a union.